packages/MarkdownEditor-Core.package/MarkdownCodeBlock.class/class/matches..st

```diff
@@ -0,0 +1,5 @@
+testing
+matches: aString
+
+	^ (aString beginsWith: '    ') not
+		and: [aString matchesRegex: ' *```+[^`]*']
```

Comment:
for someone who is not familiar with how these regexes work, its a bit hard to understand, why this one matches to backticks with an info string, but not to backticks with an info string with backticks, but I doubt a comment would do any more good here. Maybe you have an idea but if you don't, you can also just leave it like this.

Reply:
Like discussed face-to-face, we should always favour a simple solution over a regex. But regarding how the method looked before, a regex would is more understandable.
(And we can assume that people know the `[^a]` syntax)